packages/block-editor/src/components/tool-selector/index.js

```diff
@@ -44,8 +44,7 @@ function ToolSelector( props, ref ) {
 		<Dropdown
 			renderToggle={ ( { isOpen, onToggle } ) => (
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					size="compact"
```

Comment:
In the app, the `size="compact"` is already passed from the consumer, but let's add it to the component itself as well.

<img src="https://github.com/user-attachments/assets/6134575b-5fff-4718-9d84-8f2f7e9f1ac6" alt="Tool selector button" width="304">